src/core/qgscoordinatereferencesystem.cpp

```diff
@@ -72,6 +73,23 @@ QHash< long, QgsCoordinateReferenceSystem > QgsCoordinateReferenceSystem::sSrsId
 QReadWriteLock QgsCoordinateReferenceSystem::sCrsStringLock;
 QHash< QString, QgsCoordinateReferenceSystem > QgsCoordinateReferenceSystem::sStringCache;
 
+
+#if PROJ_VERSION_MAJOR>=6
+QString getFullProjString( PJ *obj )
+{
+  // see https://lists.osgeo.org/pipermail/proj/2019-May/008565.html, it's not sufficient to just
+  // use proj_as_proj_string
+  QgsProjUtils::proj_pj_unique_ptr boundCrs( proj_crs_create_bound_crs_to_WGS84( QgsProjContext::get(), obj, nullptr ) );
```

Comment:
there's a theoretical risk that proj_crs_create_count_crs_to_WGS84 might return nullpr as error, in which case you might probably use proj_as_proj_string() on the passed obj

Reply:
Thanks, fixed in https://github.com/qgis/QGIS/pull/9994